Deliverable 1/API Examples/reportSearchResults.json

```diff
@@ -0,0 +1,17 @@
+{
+  "UI_UX_Request": {
+    "raw_query": "RPI Professors",
+    "processed_query": "Rensselaer Polytechnic Institute Professors",
```

Comment:
If Querying sends UI/UX the processed query as well, we can get this info to go directly to ranking.

It is good for them as well